tests/legacy-cli/e2e/tests/i18n/ivy-localize-dl-xliff2.ts

```diff
@@ -41,6 +41,10 @@ export async function executeTest() {
     await expectFileToMatch(`${outputPath}/vendor-es5.js`, '.ng.common.locales');
     await expectFileToMatch(`${outputPath}/vendor-es2015.js`, '.ng.common.locales');
 
+    // Verify the locale data is browser compatible
+    await expectToFail(() => expectFileToMatch(`${outputPath}/vendor-es5.js`, /\bconst\b/));
+    await expectFileToMatch(`${outputPath}/vendor-es2015.js`, /\bconst\b/);
+
```

Comment:
Should you also have a test that proves that the comments have been removed?

Reply:
Added.